test/conftest.py

```diff
@@ -87,9 +87,9 @@ def workers(hook):
 
     # Reset the hook and the local worker
     syft.local_worker.clear_objects()
-    syft.frameworks.torch.hook.hook_args.hook_method_args_functions = {}
-    syft.frameworks.torch.hook.hook_args.hook_method_response_functions = {}
-    syft.frameworks.torch.hook.hook_args.get_tensor_type_functions = {}
+    syft.generic.frameworks.hook.hook_args.hook_method_args_functions = {}
```

Comment:
Can you add an import line to make those 3 lines more readable?

Reply:
I wasn't too sure about python's scoping when you import objects from modules vs. modules -- will give it a shot